packages/paste-website/src/pages/patterns/object-details/index.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Object details
-description: Use the object details pattern to show a read-only page or panel of information.
+description: The Object Details pattern displays a read-only page or panel containing object attributes.
```

Comment:
NIT: This description and the description for "notifications and feedback" do not match the previously established patten of "allows the user to", "provides the user with", etc. Should we align them, or is it not relevant for these two?

Reply:
It's been tricky to stick to one format for the Patterns since they're all so different, but imo the format of "The [ ] Pattern does [ ]" is consistent enough